ios/test_app.rb

```diff
@@ -142,15 +144,37 @@ def make_project!(xcodeproj, project_root, target_platform)
   FileUtils.cp(File.join(src_xcodeproj, 'project.pbxproj'), dst_xcodeproj)
   FileUtils.ln_sf(File.join(src_xcodeproj, 'xcshareddata'), dst_xcodeproj)
 
+  # Copy Xcode build configuration files
+  common_xcconfig = project_path('ReactTestApp.common.xcconfig', target_platform)
+  if File.exist?(common_xcconfig)
+    FileUtils.cp(common_xcconfig, destination)
+    %w[debug release].each do |config|
+      xcconfig = project_path("ReactTestApp.#{config}.xcconfig", target_platform)
+      next unless File.exist?(xcconfig)
+
+      dst_xcconfig = File.expand_path("ReactTestApp.#{config}.xcconfig", destination)
+      File.open(dst_xcconfig, 'w') do |fo|
+        pods_xcconfig_path = File.join(
+          project_root.relative_path_from(destination).to_s,
+          'Pods',
+          'Target Support Files',
+          'Pods-ReactTestApp',
+          "Pods-ReactTestApp.#{config}.xcconfig"
+        )
```

Comment:
@alloy: Is this path stable? Is there a better way to get to the `.xcconfig` files generated by CocoaPods?

Reply:
Yeah that should be stable enough 👍